accounts/src/main/java/eu/kevin/accounts/bankselection/providers/DefaultCountryIsoProvider.kt

```diff
@@ -0,0 +1,22 @@
+package eu.kevin.accounts.bankselection.providers
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.telephony.TelephonyManager
+import eu.kevin.common.extensions.getCurrentLocale
+
+internal class DefaultCountryIsoProvider(
+    private val context: Context,
+    private val telephonyManager: TelephonyManager
+) {
+    fun getDefaultCountryIso(): String {
+        if (
+            context.packageManager.hasSystemFeature(PackageManager.FEATURE_TELEPHONY) &&
```

Comment:
For `networkCountryIso` the docs actually propose checking for `FEATURE_TELEPHONY_RADIO_ACCESS`
![image](https://user-images.githubusercontent.com/11537045/236133808-149503ef-8095-4b6f-b35d-f91c7c0a638b.png)

Reply:
Added `FEATURE_TELEPHONY_SUBSCRIPTION` instead, as it's required for `simCountryIso`